.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
`.idea/` implies all items under that directory
```suggestion
*.idea
.idea/
```

Reply:
That's what I want :)